README.md

```diff
@@ -231,7 +231,7 @@ The initialize method takes the following arguments:
 
 ### instanceId
 
-As of version 5.0.0, `instanceId` is now automatically generated.
```

Comment:
Scouting: Noticed this was wrong previously. We're already in v5, so next major is 6.